### ISSUE TITLE
Fails with "error: Flatpak system operation ConfigureRemote not allowed for user"

### ISSUE THREAD

**Issue by Mek101** [refi64/stylepak#11]
```
~ >>> pakitheme install-system                                                                                                                                                                                                            [1]
Converting theme: Adwaita-maia
Found theme located at: /usr/share/themes/Adwaita-maia
Processing application org.gtk.Gtk3theme.Adwaita-maia
Saving AppStream /home/mek101/.cache/pakitheme/Adwaita-maia/build/files/share/app-info/xmls/org.gtk.Gtk3theme.Adwaita-maia.xml.gz
Done!
3100a584f7f93b27dbe73791eccbb31445f385e030fc32c3188ab72bbdb6a57b
96cb548414c5ccfa97afa5dc050fc5286080ccafe921d524ef8e309125eae1fc
error: Flatpak system operation ConfigureRemote not allowed for user
```

Running it again with `PAKITHEME_VERBOSE=1 pakitheme install-system` ends in
```
...
Processing application org.gtk.Gtk3theme.Adwaita-maia
Saving AppStream /home/mek101/.cache/pakitheme/Adwaita-maia/build/files/share/app-info/xmls/org.gtk.Gtk3theme.Adwaita-maia.xml.gz
Done!
+ ostree --repo=/home/mek101/.cache/pakitheme/Adwaita-maia/repo commit -b base --tree=dir=/home/mek101/.cache/pakitheme/Adwaita-maia/build
9186ce2ef09a675c40e91a3028a266260c9576989fe9f47aa1dc33898ebda50b
+ bundles=()
+ read -r arch
++ flatpak list --runtime --columns=arch:f
++ sort -u
+ bundle=/home/mek101/.cache/pakitheme/Adwaita-maia/org.gtk.Gtk3theme.Adwaita-maia-x86_64.flatpak
+ rm -rf /home/mek101/.cache/pakitheme/Adwaita-maia/build
+ ostree --repo=/home/mek101/.cache/pakitheme/Adwaita-maia/repo checkout -U base /home/mek101/.cache/pakitheme/Adwaita-maia/build
+ read -rd '' metadata
+ :
+ echo -n '[Runtime]
name=org.gtk.Gtk3theme.Adwaita-maia
runtime=org.gtk.Gtk3theme.Adwaita-maia/x86_64/3.22
sdk=org.gtk.Gtk3theme.Adwaita-maia/x86_64/3.22'
++ cat /home/mek101/.cache/pakitheme/Adwaita-maia/build/metadata
+ ostree --repo=/home/mek101/.cache/pakitheme/Adwaita-maia/repo commit -b runtime/org.gtk.Gtk3theme.Adwaita-maia/x86_64/3.22 --add-metadata-string 'xa.metadata=[Runtime]
name=org.gtk.Gtk3theme.Adwaita-maia
runtime=org.gtk.Gtk3theme.Adwaita-maia/x86_64/3.22
sdk=org.gtk.Gtk3theme.Adwaita-maia/x86_64/3.22' --link-checkout-speedup /home/mek101/.cache/pakitheme/Adwaita-maia/build
babde0fddd4c2e4a765dc6d7fecbdce522e07b26dd0c92356d99c367e4196809
+ flatpak build-bundle --runtime /home/mek101/.cache/pakitheme/Adwaita-maia/repo /home/mek101/.cache/pakitheme/Adwaita-maia/org.gtk.Gtk3theme.Adwaita-maia-x86_64.flatpak org.gtk.Gtk3theme.Adwaita-maia 3.22
+ trap 'rm "$bundle"' EXIT
+ bundles+=("$bundle")
+ read -r arch
+ for bundle in "${bundles[@]}"
+ flatpak install -y --system /home/mek101/.cache/pakitheme/Adwaita-maia/org.gtk.Gtk3theme.Adwaita-maia-x86_64.flatpak
error: Flatpak system operation ConfigureRemote not allowed for user
+ rm /home/mek101/.cache/pakitheme/Adwaita-maia/org.gtk.Gtk3theme.Adwaita-maia-x86_64.flatpak
```

**Comment by Mek101** [comment]
Nvm: found the issue: it happens if the user doesn't have the permissions, however it can be fixed by removing the `-y` option from flatpak, which will make so that it will prompt the user for authorization via password

**Comment by refi64** [comment]
Well that...kind of sucks, I wonder if this is supposed to be intentional.

**Comment by refi64** [comment]
Should be fixed by #16, please try it and let me know.
